app/config/errors.php

```diff
@@ -10,61 +10,64 @@
     /** General Errors */
     Exception::GENERAL_UNKNOWN => [
         'name' => Exception::GENERAL_UNKNOWN,
-        'description' => 'An unknown error has occured. Please check the logs for more information.',
+        'description' => 'An unknown error has occured. Please submit a support ticket at [TODO: Where do people get help?]() with reproduction steps for further assistance.',
```

Comment:
Is there a places we want to direct users who need support? Maybe an email address?

Reply:
This is kind of a catch-all exception. I'd like to see a GitHub issue for this with:

1. reproduction steps
2. if self-hosting, logs from the appwrite container
3. if on cloud, project ID so we can try to lookup the logs

Then, we can look into how to catch the generic exception and return an better exception.

Reply:
Sounds good,